src/components/navbar/navbar.component.tsx

```diff
@@ -87,7 +88,9 @@ const Navbar: React.FC<NavbarProps> = ({
                 <Logo />
               </HeaderLink>
               <HeaderGlobalBar>
-                <ExtensionSlot extensionSlotName="top-nav-actions-slot" />
+                <div className={styles.headerGlobalActionSlot}>
```

Comment:
@vasharma05  i see wrapping the Extension slot  in this div renders things well  cc @FlorianRappl

Reply:
@vasharma05  can you go on to add a ticket Id to your description and the commit message!

Reply:
> @vasharma05  can you go on to add a ticket Id to your description and the commit message!

Yeah sure, I'll add the details.

Reply:
The extra div is not needed. Just apply the class name to the extension slot. I guess you rather want the flex to apply internally - no need for the extra div.

Reply:
Didn't know about it before. @nk183 is working on it at #66 , so I think I should close this one.